### ISSUE TITLE
Render Elements in Tabset

### ISSUE THREAD

**Issue by Fabi1992** [Appsilon/shiny.semantic#39]
Started to learn about this great package and was not able to render elements in **tabset**.

UI.R:

```
library(shiny)
shinyUI(semanticPage(
  suppressDependencies("bootstrap"),
  tabset(tabs=
  list(
    list(menu="First Tab", content=
  "Text works well"),
   list(menu="Second Tab",content=plotOutput("plot1"))
  )
)
  ))
```
Server.R:

```
library(shiny)
library(shiny.semantic)
library(highlighter)

shinyServer(function(input, output) {
  output$plot1 <- renderPlot(plot(1,1))

})
```
Plot (and other render elements) are not appearing in tabset.

**Comment by krystian8207** [comment]
Hi. We added [37bfd80](https://github.com/Appsilon/shiny.semantic/commit/37bfd800983430fd2f33566c9f1b802ee4ddb093) fix to this issue. To proper functioning, you should also add suspendWhenHidden = FALSE outputOption for each output, as in the following example:
```
library(shiny)
library(shiny.semantic)
library(highlighter)
library(magrittr)
library(plotly)

ui <- shinyUI(semanticPage(
  suppressDependencies("bootstrap"),
  tabset(tabs=
           list(
             list(menu="First Tab", content= "Text works well"),
             list(menu="Second Tab",content=plotOutput("plot1")),
             list(menu="Second Tab",content=plotlyOutput("plot2"))
           )
  )
  # Below comment fixes issue manually, without last commit:
  # tags$script(paste0("$('.menu .item').tab({onVisible: function() {$(window).resize()} });"))
))
server <- shinyServer(function(input, output) {
  output$plot1 <- renderPlot(plot(1,1))
  outputOptions(output, "plot1", suspendWhenHidden = FALSE)

  output$plot2 <- renderPlotly(plot_ly(mtcars, x=~mpg, y=~cyl, width = "100%"))
  outputOptions(output, "plot2", suspendWhenHidden = FALSE)
})

shiny::shinyApp(ui, server)
```
If you want to use cran version of our package just uncomment the last parameter of ui above, otherwise install the package from github.

**Comment by biranjan** [comment]
First, thanks for such a great package it really adds shine to shiny. For some reason when I reinstalled the package via github it didn't really work, however, the solution presented above did work. There seem to be missing: **,** before the tags$script though in case if anyone copy pastes it without adding missing **,**.

**Comment by filipstachura** [comment]
@biranjan Thanks, for the feedback.

Closing the issue.

**Comment by maju116** [comment]
@biranjan There's no more need to add `suspendWhenHidden = FALSE` in `outputOptions`. This issue was solved in [86b274d7e9e82b8e009e4fabd1a3799a45ea98b3]
